libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -402,6 +402,11 @@ export const estateMemberMapper = (estateRaw: Erfingar): EstateMember => {
     advocate: estateRaw.malsvari ? mapAdvocate(estateRaw.malsvari) : undefined,
     email: estateRaw.netfang ?? '',
     phone: estateRaw.simi ?? '',
+    // We can assume that heir is a foreign citizen if national id is missing or ends with '0000'
+    foreignCitizenship:
+      !estateRaw.kennitala || estateRaw.kennitala.endsWith('0000')
```

Comment:
same as above

Reply:
> is there really not already a utility method in monorepo that can check this instead of having to stitch this logic on the fly ?

Could use isPerson() from kennitala module?, can that be used in a client library?